sdk/build.gradle

```diff
@@ -29,6 +29,10 @@ android {
         disable 'InvalidPackage' // okio
     }
 
+    androidExtensions {
+        experimental = true
```

Comment:
Experimental? When will the feature we use is out of the experimental phase?

Reply:
It's been flagged as `experimental` for a while now, about a year? @nuxzero correct me if I'm wrong?

Reply:
Sorry, I don't what actually when it will release to stable version but I think they will not release in soon. 

Q: Should we use it?
A: I remember there is an important note about using experimental mode but it gone from the website. So, case of building the SDK I think it quite risky to the SDK because it might raise to bugs in a head since we need to support a ton of variants (e.g. Java, Kotlin) that I'm quite concern. But we have test enough to cover all areas that we use it that might acceptable. 😂

Reply:
I feel it's fine to use, but again we can keep testing it to see if there are any issues. If there are, it's not that hard to undo the changes.